src/models/m_category.erl

```diff
@@ -563,7 +560,7 @@ name_to_id(Name, Context) when is_atom(Name); is_binary(Name); is_list(Name) ->
     end.
 
 %% @doc Map a category name to an id, be flexible with the input
--spec name_to_id_check(binary()|list()|integer()|{integer()}, #context{}) -> integer().
+-spec name_to_id_check(atom()|binary()|list()|integer()|{integer()}, #context{}) -> integer().
```

Comment:
The spec in line 563 uses a record, please define a type for the record and use that instead.